contrib/auto-render/test/auto-render-spec.js

```diff
@@ -191,44 +204,68 @@ describe("A delimiter splitter", function() {
             ]);
     });
 
-    it("works with more than one start datum", function() {
-        const startData = [
-            {type: "text", data: "hello ( world ) boo"},
-            {type: "math", data: "math", rawData: "(math)", display: true},
-            {type: "text", data: "hello ( world ) boo"},
-        ];
-
-        expect(splitAtDelimiters(startData, "(", ")", false)).toEqual(
+    it("handles nested delimiters irrespective of order", function() {
+        expect(splitAtDelimiters("$\\fbox{\\(hi\\)}$",
             [
-                {type: "text", data: "hello "},
-                {type: "math", data: " world ",
-                    rawData: "( world )", display: false},
-                {type: "text", data: " boo"},
-                {type: "math", data: "math", rawData: "(math)", display: true},
-                {type: "text", data: "hello "},
-                {type: "math", data: " world ",
-                    rawData: "( world )", display: false},
-                {type: "text", data: " boo"},
+                                     {left:"\\(", right:"\\)", display:false},
+                                     {left:"$", right:"$", display:false},
```

Comment:
style nit: indentation

Reply:
weird, I thought I'd run `--whitespace=fix`. am I missing something?

Reply:
What command are you passing `--whitespace=fix` to?

Reply:
`git rebase`. what is the canonical way to fix indentation in KaTeX code?

Reply:
emacs `untabify` and `yarn eslint . --fix` seem to do the trick

Reply:
I didn't know about that git command.  I was going to setup prettier at one point but there's a bunch of code/comments in this repo that prettier ends up making worse.